pyperf/pyperf_run

```diff
@@ -245,15 +252,11 @@ done
 
 if [ $to_pbench -eq 0 ]; then
 	rm -rf pyperformance
-	git clone https://github.com/python/pyperformance
+	python3 -m pip install pyperformance==$PYPERF_VERSION
```

Comment:
We should be making sure that `pip` is installed.  Either through `python3 -m ensurepip --upgrade` or by specifying `python3-pip` as a dependency.